src/commands/api-mesh/source/get.js

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { Command, Flags } = require('@oclif/core');
+const SourceRegistryStorage = require('source-registry-storage-adapter');
+const { promptMultiselect, promptSelect, promptConfirm, initRequestId } = require('../../../helpers');
+const ncp = require('node-clipboardy');
+const chalk = require('chalk');
+const config = require('@adobe/aio-lib-core-config');
+const logger = require('../../../classes/logger');
+
+class GetCommand extends Command {
+	constructor() {
+		super(...arguments);
+		this.sourceRegistryStorage = new SourceRegistryStorage(
+			config.get('api-mesh.sourceRegistry.path'),
+		);
+	}
+
+	async run() {
+		await initRequestId();
```

Comment:
Please wrap the whole run function in a try-catch. If something in the function fails, the customer will be left with error stack that might not make sense to them. Please wrap it in a try-catch and add logger.error inside so we can get error stack information if needed while debugging.